src/components/Navbar/Navbar.component.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import MenuIcon from '@material-ui/icons/Menu';
+import CloseIcon from '@material-ui/icons/Close';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Switch from '@material-ui/core/Switch';
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import './Navbar.styles.css';
+import IconButton from '@material-ui/core/IconButton';
+import SearchIcon from '@material-ui/icons/Search';
+import InputBase from '@material-ui/core/InputBase';
+
+const CustomSwitch = withStyles({
+  switchBase: {
+    color: '#fff',
+  },
+  checked: {
+    color: '#f2575d',
+  },
+  track: {
+    backgroundColor: '#f2575d',
+  },
+})(Switch);
+
+const useStyles = makeStyles({
```

Comment:
Here you are using one approach to apply styles but you are also defining more styles in the file `Navbar.styles.css`, using only one approach is better. Also, as part of the challenge you should use `styled-components` at least in one component, you can create one in future challenges